sdks/java/core/src/main/java/org/apache/beam/sdk/util/SystemDoFnInternal.java

```diff
@@ -26,9 +26,6 @@
 /**
  * Annotation to mark {@code DoFns} as an internal component of the Beam SDK.
  *
- * <p>Currently, the only effect of this is to mark any aggregators reported by an annotated
- * {@code DoFn} as a system counter (as opposed to a user counter).
- *
  * <p>This is internal to the Beam SDK.
  */
 @Documented
```

Comment:
It's used in several places to annotate DoFns. I figured there may be a reason that is not only related to aggregators.